Packs/Tenable_io/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Tenable.io
+- Fixed an issue in ***tenable-io-get-vulnerabilities-by-asset***.
+- Fixed the *raw-response* argument for all commands.
```

Comment:
We should note all the commands that were added

Reply:
NOT STARTED
Update 02.23.2021 1034 HRS EST: